game-core/src/main/java/games/strategy/engine/message/unifiedmessenger/UnifiedMessenger.java

```diff
@@ -117,7 +119,12 @@ private RemoteMethodCallResults invokeAndWaitRemote(final RemoteMethodCall remot
     final Invoke invoke = new HubInvoke(methodCallId, true, remoteCall);
     send(invoke, messenger.getServerNode());
 
-    Interruptibles.await(latch);
+    if (!Interruptibles.awaitResult(() -> latch.await(1, TimeUnit.MINUTES)).result.orElse(false)) {
```

Comment:
Is 1 minute potentially an over-generous timeout?

Have you simulated this error by any chance? Did it show the option to submit an error report (I'm hoping yes)

Reply:
I haven't tested if there is an error dialog, but given that a couple of lines below a similar exception is thrown, hopes are that we'll get an error dialog.
But I can test this tomorrow.

Reply:
And the 1 minute is arbitrary, open for suggestions

Reply:
20 seconds?